### DISCUSSION TITLE
Details on Problem

### DISCUSSION THREAD

**Discussion by kumarsiddappa-git** [mrdbourke/pytorch-deep-learning#744] [Q&A]
In the moon_cirlce or circle , Y indicates the label weather its red or blue, but what does the columns in the X indicates ? if we are using linear regression , the X has two columns which are X1 and X2 are placed in the equation y=w1X1+w2X2+B and predict Y , what are we predicting here ? we do not have Y values apart from 0 and 1 classes ? could you please the problem statement once more

**Comment by sametucuncu** [comment]
I think what we are predicting in this problem is whether X value belongs to red circle(0) or blue circle(1) which indicates Y. Because the problem is classification problem, specifically binary classification, we only predict a class of the X value, 0 or 1. Do not confuse it with regression problem which we are trying to predict actual value of output.

**Reply by shisirkha** [reply]
Correct
